core/lib/spree/app_configuration.rb

```diff
@@ -42,6 +42,12 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Integer] Number of products to display in admin (default: +10+)
     preference :admin_products_per_page, :integer, default: 10
 
+    # @!attribute [rw] admin_themes
+    #   @return [Hash] A hash containing the themes that are available for the admin panel
+    preference :admin_themes, :hash, default: {
```

Comment:
What do you think if we define those new settings in `Spree::BackendConfiguration`? I know that we have other `admin` related settings in `Spree::Config`, but we also have some of them in the backend file, and it seems to make more sense. Thoughts?